src/roiextractors/extractors/inscopixextractors/inscopiximagingextractor.py

```diff
@@ -23,6 +26,12 @@ def __init__(self, file_path: PathType):
         file_path : PathType
             Path to the Inscopix file.
         """
+        if sys.version_info < (3, 9) or sys.version_info >= (3, 13):
```

Comment:
We don't support python 3.9, the first condition is necessary.

Also, can you create a (local) variable named `python_version = sys.version_info`? This will make reading the code easier and faster for people who don't know what sys_version_info stands for.